python/legion_linux/legion_linux/legion.py

```diff
@@ -1381,7 +1381,7 @@ def __init__(self, expect_hwmon=True, use_legion_cli_to_write=False, config_dir=
         self.lockfancontroller = LockFanController()
         self.rapid_charging = RapidChargingFeature(None)
         self.battery_conservation = BatteryConservation(None)
-        # TOOD: fix this by resolving ciruclar depedency by facade or similar
+        # TODO: fix this by resolving circular dependency by facade or similar
```

Comment:
You need to remove TODO: on this line
See this comment https://github.com/johnfanv2/LenovoLegionLinux/pull/111#issuecomment-1750443039

Reply:
I think i can edit the code myself let me check

Reply:
I solve it